cardano-testnet/src/Testnet/Defaults.hs

```diff
@@ -515,6 +517,18 @@ defaultDRepSkeyFp n = "drep-keys" </> ("drep" <> show n) </> "drep.skey"
 defaultDRepKeyPair :: Int -> PaymentKeyPair
 defaultDRepKeyPair n = PaymentKeyPair (defaultDRepVkeyFp n) (defaultDRepSkeyFp n)
 
+-- | The relative path to SPO key pairs in directories created by cardano-testnet
+defaultSPOKeys :: Int -> PoolNodeKeys
+defaultSPOKeys n =
```

Comment:
Why the paths are different than those in `Testnet.Start.Cardano`? Can we reuse it there?